.github/workflows/nix-build.yml

```diff
@@ -0,0 +1,14 @@
+name: "Test"
```

Comment:
Maybe renaming this to something more explicit and nix related?
(eg. "Nix packaging test" / "packaging/nix packaging test"...)